### PR TITLE
fix black warnings; update for yaml issue; bump version; add 3.9 to ci

### PR DESCRIPTION
**Pull request by mkinney**

### CODE REVIEW

**Comment by mkinney** [comment]
@albertodonato @bjornt Do you want to add me to the maas project so I can help out?

**Comment by mkinney** [comment]
Also, can you add me to the pypi group, too? I want to push some releases to pypi. (alternatively, setup PYPI_API_TOKEN in this repo so we can push from a github action)

See https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/ )

File: setup.py [hunk 1]
```diff
@@ -36,6 +36,8 @@ def read(filename):
         'Programming Language :: Python :: 3 :: Only',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```
Comment (albertodonato) [inline setup.py:40]: Could  you please add 3.9 as well?

Comment (mkinney) [reply setup.py:39]: Sure

**Comment by albertodonato** [comment]
Thanks for the contribution.
Could you please sign the contributor agreement (https://ubuntu.com/legal/contributors), so that your changes can be merged?

**Comment by mkinney** [comment]
Cool. CI passes on 3.6 thru 3.9.

File: setup.py [hunk 2]
```diff
@@ -27,15 +27,17 @@ def read(filename):
     author='MAAS Developers',
     author_email='maas-devel@lists.launchpad.net',
     url='https://github.com/maas/python-libmaas',
-    version="0.6.6",
+    version="0.6.7",
```
Comment (albertodonato) [inline setup.py:30]: Could you please revert the version bump? We usually do it as a separate commit before releasing.

Comment (mkinney) [reply setup.py]: Done

**Review by albertodonato — APPROVED** [review]
